.github/workflows/publish.yaml

```diff
@@ -11,8 +11,27 @@ jobs:
   test:
     uses: ./.github/workflows/main.yaml
 
+  # First create the release so that we have somewhere to upload the distribution zips to
+  create-release:
+    runs-on: ubuntu-latest
+    permissions:
```

Comment:
Dropped `packages` here as we don't need that to publish a release, just to upload to GH packages